### PR TITLE
Added five new locales to flutter_localizations

### PR DESCRIPTION
**Pull request by HansMuller**
Add support for five new locales:

ko - Korean
nl - Dutch
pl - Polish
th - Thai
tr - Turkish

@yegor: I added scriptCategory and timeOfDayFormat entries to the new locales' arb files:

packages/flutter_localizations/lib/src/l10n/material_ko.arb
packages/flutter_localizations/lib/src/l10n/material_nl.arb
packages/flutter_localizations/lib/src/l10n/material_pl.arb
packages/flutter_localizations/lib/src/l10n/material_th.arb
packages/flutter_localizations/lib/src/l10n/material_tr.arb

They new entries are guesses, please take a look.

### CODE REVIEW

**Comment by HansMuller** [comment]
This PR should land before https://github.com/flutter/flutter/pull/13653 so that #13653 can incorporate the new locales.

File: packages/flutter_localizations/lib/src/l10n/material_ko.arb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "scriptCategory": "dense",
+  "timeOfDayFormat": "ah:mm",
```
Comment (yjbanov) [inline packages/flutter_localizations/lib/src/l10n/material_ko.arb]: In our system these strings only affect the layout of the time picker, but _technically_ Korean format has one space between "a" and "h:mm", i.e. "a h:mm" (see http://demo.icu-project.org/icu-bin/locexp?d_=en&_=ko_KR)

File: packages/flutter_localizations/lib/src/l10n/material_ko.arb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "scriptCategory": "dense",
```
Comment (yjbanov) [inline packages/flutter_localizations/lib/src/l10n/material_ko.arb]: You can double-check the script categories on this page: https://material.io/guidelines/style/typography.html#typography-language-categories-reference

**Review by yjbanov — APPROVED** [review]

**Comment by yjbanov** [comment]
lgtm

File: packages/flutter_localizations/lib/src/l10n/material_ko.arb (hunk 2 above)
Comment (HansMuller) [reply packages/flutter_localizations/lib/src/l10n/material_ko.arb]: I've checked them and changed Thai (th) to 'tall'.

File: packages/flutter_localizations/lib/src/l10n/material_ko.arb (hunk 1 above)
Comment (HansMuller) [reply packages/flutter_localizations/lib/src/l10n/material_ko.arb]: OK, fixed.

**Comment by HansMuller** [comment]
I updated flutter_localizations/lib/src/l10n/README.md to explain how scriptCategory and timeOfDayFormat are defined.

**Pull request merged**